rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +86,21 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self._args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell')
+        Shell.add_arguments(parser)
+        return parser.parse_args(argv)
+
+    @staticmethod
+    def add_arguments(parser):
+        group = parser.add_argument_group('Options for rqt_shell plugin')
+        group.add_argument('-i', '--init_script',
+                            help="load this script after startup in $SHELL")
```

Comment:
The term `Sources this script ...` is incorrect in this context. When you source a script all the variables as well as functions become available. What you describe (requiring `export`) is "running" / "executing" therefore the help should start with `Executes the script ...`.